### PR TITLE
[WIP][3.0][Form] Removed usage of deprecated form tag attributes

### PR DESCRIPTION
**Pull request by wouterj**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | yes |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

WIP because it needs some adjustments after https://github.com/symfony/symfony/pull/15926 is merged

### CODE REVIEW

**Comment by fabpot** [comment]
#15926 has been merged now. @WouterJ Can you rebase and update.

**Comment by Tobion** [comment]
@fabpot 2.8 needs to be merged into master first. Are you on it?

**Comment by fabpot** [comment]
@Tobion No, I'm in a meeting right now, can you do it?

**Comment by Tobion** [comment]
@fabpot merged

**Comment by WouterJ** [comment]
Rebased & included rename alias -> extended_type

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/FormPass.php [hunk 1]
```diff
@@ -46,12 +46,8 @@ public function process(ContainerBuilder $container)
         foreach ($container->findTaggedServiceIds('form.type_extension') as $serviceId => $tag) {
             if (isset($tag[0]['extended_type'])) {
                 $extendedType = $tag[0]['extended_type'];
-            } elseif (isset($tag[0]['alias'])) {
-                @trigger_error('The alias option of the form.type_extension tag is deprecated since version 2.8 and will be removed in 3.0. Use the extended_type option instead.', E_USER_DEPRECATED);
-                $extendedType = $tag[0]['alias'];
             } else {
-                @trigger_error('The extended_type option of the form.type_extension tag is required since version 2.8 and will be removed in 3.0.', E_USER_DEPRECATED);
-                $extendedType = $serviceId;
+                throw new \InvalidArgumentException(sprintf('Tagged form type extension must have the extended type configured using the extended_type/extended-type attribute, none was configured for the "%s" service.', $serviceId));
```
Comment (Tobion) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/FormPass.php]: "Tagged form type extensionS"

**Comment by fabpot** [comment]
Thank you @WouterJ.

**Pull request merged**